google/cloud/aiplatform/datasets.py

```diff
@@ -65,28 +64,13 @@ def __init__(
                 credentials set in aiplatform.init.
         """
 
-        # Fully qualified dataset name, i.e. "projects/.../locations/.../datasets/12345"
-        valid_name = extract_fields_from_resource_name(
-            resource_name=dataset_name, resource_noun="datasets"
+        dataset_name = full_name(
```

Comment:
Please add this to the same logic to the Model class. Unless you plan to add it to the open Endpoint PR.

Reply:
will put it in the endpoint pr